src/components/posts/PostText.js

```diff
@@ -2,20 +2,20 @@ import React from "react";
 import "./posts.css";
 import "../mainContent/mainContent.css";
 export default function PostText(props) {
-  /*This mapping function takes the array of post body Text and renders each paragraph as a line of
-body text */
-  return (
-    <div>
-      {props.bodyText.map((paragraph, index) => {
-        let postClass = `post-body-text body-text ${
-          paragraph.bold ? "text-bold" : ""
-        } `;
-        return (
-          <div key={index} className={postClass}>
-            {paragraph.body}
-          </div>
-        );
-      })}
-    </div>
-  );
+  /*This mapping function takes the array of post body Text and renders each paragraph that
+  should be designated as visible */
+  let visibleParagraphs = props.bodyText.map((paragraph, index) => {
+    let postClass = `post-body-text body-text ${
+      paragraph.bold ? "text-bold " : ""
+    } `;
+    postClass += `${props.currentVisibleText < index + 1 ? "hidden-text" : ""}`;
```

Comment:
Why is this in a string comprehension block (the `${}` thing) - it seems like it could just be `props.currentVisibleText < index + 1 ? "hidden-text" : ""` 

Also, you could just add it to the `postClass` original literal!